addons/mrp/report/mrp_report_bom_structure.py

```diff
@@ -19,7 +19,7 @@ def get_html(self, bom_id=False, searchQty=1, searchVariant=False):
 
     @api.model
     def get_warehouses(self):
-        return self.env['stock.warehouse'].search_read([('company_id', 'in', self.env.companies.ids)], fields=['id', 'name'])
+        return self.env['stock.warehouse'].with_context(active_test=False).search_read([('company_id', 'in', self.env.companies.ids)], fields=['id', 'name'])
```

Comment:
```suggestion
        return self.env['stock.warehouse'].with_context(active_test=False).search_read([
                        ('company_id', 'in', self.env.companies.ids)
                    ], fields=['id', 'name'])
```
Avoid more than 100 characters in a single line.